<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pyBath</class>
 <widget class="QMainWindow" name="pyBath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>627</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>783</width>
    <height>627</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Immersion bath</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>temperature-2-xxl.png</normaloff>temperature-2-xxl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Medium Properties:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_heat_capacity_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Heat Capacity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="_heat_capacity_spin"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_mass_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Mass:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="_mass_spin"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tank Properties:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="_emissivity_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Emissivity:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="_emissivity_spin"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_area_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Area:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="_area_spin"/>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Element Properties:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_resistance_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Resistance:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="_resistance_spin"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_voltage_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Voltage:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="_voltage_spin"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Control coefficients:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="_p_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>P term:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="_p_spin"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_i_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>I term</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="_i_spin"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="_d_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>D term</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="_d_spin"/>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Task Parameters:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="_monitor_check">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Monitor mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_target_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Target Temperature:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="_target_spin"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="_btn_start">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start Process</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="_btn_stop">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Stop Process</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="_state_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Current State:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="_state_out_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="_ambient_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Ambient Temp:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="_env_label">
        <property name="font">
         <font>
          <family>NanumBarunGothic</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="_bath_label">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
         </font>
        </property>
        <property name="text">
         <string>Medium Temp:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="_bath_label_2">
        <property name="font">
         <font>
          <family>NanumBarunGothic</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="_save_config"/>
    <addaction name="_load_config"/>
    <addaction name="_quit"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="_plot_dock">
   <property name="minimumSize">
    <size>
     <width>783</width>
     <height>363</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="InteractiveGRWidget" name="_stage" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>765</width>
         <height>320</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="_save_config">
   <property name="text">
    <string>Save Config...</string>
   </property>
  </action>
  <action name="_load_config">
   <property name="text">
    <string>Load Config...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveGRWidget</class>
   <extends>QWidget</extends>
   <header>qtgr</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
